package.json

```diff
@@ -53,10 +53,10 @@
     "pug": "^2.0.0-rc.4",
     "rimraf": "^2.6.2",
     "run-with-ganache": "^0.1.1",
-    "solc": "^0.4.23",
+    "solc": "^0.4.24",
```

Comment:
should update `pragma` on source files.

Reply:
Will update the pragma on other pr along fixing the warnings which popped up from 0.4.24.

Reply:
see pr #477